packages/crayons-core/src/components/country-phone/config.js

```diff
@@ -0,0 +1,9 @@
+import countries from './countries.json';
```

Comment:
There seems to be no configuration in the file. Can you kindly consider moving the import to the component?

Reply:
Fixed : ✅